### ISSUE TITLE
apidiff should ignore InternalCall attribute

### ISSUE THREAD

**Issue by jaykrell** [mono/mono#9559]
## Steps to Reproduce

1. Submit a PR in which public APIs do not change, but InternalCall is added or removed. Such as the old InternalCall is reimplemented in C#, possibly on top of a different InternalCall.

### Current Behavior

apidiff fails, and valid changes require more coordination and process than they should.

### Expected Behavior

There should be no apidiff failure when the only change is add/removal of InternalCall attribute, as long as public/protected/internal access is unchanged.

## On which platforms did you notice this

[x] macOS
[x] Linux
[x]] Windows

**Comment by jaykrell** [comment]
For example https://github.com/mono/mono/pull/9503.

**Comment by marek-safar** [comment]
We'd need to filter out on `MethodImplAttribute` which should be fine as it's implementation detail even if useful to see in special cases.

Similar to #8036

**Comment by akoeplinger** [comment]
Hm, I'm a bit hesitant to ignore more stuff, this is supposed to catch unintentional mistakes so I'd say we should wait and see whether it comes up more often :)

**Comment by marek-safar** [comment]
I agree that most of the time MethodImplAttribute change is just noise, we probably only care about change to InternalCall but I think that should be caught as the extern keyword change

**Comment by jaykrell** [comment]
I don't think extern matters either.
Only access  -- public or not public, and existance.

**Comment by marek-safar** [comment]
It does when we import new external code
